packages/wallet/src/pages/index.tsx

```diff
@@ -1,5 +1,62 @@
 import { AppLayout } from '@/components/layouts/AppLayout'
+import { PageHeader } from '@/components/PageHeader'
+import { Button } from '@/ui/Button'
+import { Link } from '@/ui/Link'
+import { mockAccountList, type Account } from '@/utils/mocks'
+import {
+  type InferGetServerSidePropsType,
+  type GetServerSideProps
+} from 'next/types'
 
-export default function Home() {
-  return <AppLayout>Content</AppLayout>
+type HomeProps = InferGetServerSidePropsType<typeof getServerSideProps>
+
+export default function Home({ accounts }: HomeProps) {
+  return (
+    <AppLayout>
+      <PageHeader
+        title="Hello, John Doe!"
+        message="Here is your account overview!"
+      />
+      <div className="mt-10 flex flex-col space-y-10">
+        <div>
+          <h2 className="text-lg font-light md:text-xl">Balance</h2>
+          <p className="text-2xl font-semibold md:text-4xl">$2,934</p>
+        </div>
+        <div className="flex w-full flex-col space-y-2 md:max-w-lg">
+          <div className="flex items-center justify-between">
+            <h3 className="text-lg font-semibold">My Accounts</h3>
+            <Button intent="outline" size="sm" aria-label="manage accounts">
+              Manage Accounts
+            </Button>
+          </div>
+          <div className="flex flex-col space-y-5">
+            {accounts.map((account) => (
+              <Link
+                href={`accounts/${account.id}`}
+                key={account.id}
+                className="flex items-center justify-between rounded-md bg-brand-turqoise px-4 py-4"
```

Comment:
Looks ok, but maybe you can also try with py-3 to make it just a bit smaller, but still not crowded :)